app/client/components/VersionUpdateBanner.ts

```diff
@@ -0,0 +1,79 @@
+import {Banner, buildBannerMessage} from 'app/client/components/Banner';
+import {makeT} from 'app/client/lib/localization';
+import {localStorageJsonObs} from 'app/client/lib/localStorageObs';
+import {getGristConfig} from 'app/common/urlUtils';
+import {Computed, Disposable, dom, makeTestId, Observable} from 'grainjs';
+import { AppModel } from 'app/client/models/AppModel';
+
+const t = makeT("VersionUpdateBanner");
+const testId = makeTestId('test-version-update-banner-');
+
+interface ShowVersionUpdateBannerPrefer {
+  isShown: boolean,
+  version?: string,
+}
+
+export class VersionUpdateBanner extends Disposable {
+  private readonly _currentApp = new Observable(this._appModel);
```

Comment:
This observable is not needed,  I think this is old code from a different approche.